src/server/scripts/Kalimdor/CavernsOfTime/BattleForMountHyjal/instance_hyjal.cpp

```diff
@@ -132,128 +133,127 @@ class instance_hyjal : public InstanceMapScript
+            case NPC_ALLIANCE_KNIGHT:
+            case NPC_ALLIANCE_FOOTMAN:
+            case NPC_ALLIANCE_RIFLEMAN:
+            case NPC_ALLIANCE_PRIEST:
+            case NPC_ALLIANCE_SORCERESS:
+            case NPC_JAINA:
+                _baseAlliance.insert(creature->GetGUID());
+                break;
+                // Horde base
+            case NPC_HORDE_HEADHUNTER:
+            case NPC_HORDE_SHAMAN:
+            case NPC_HORDE_GRUNT:
+            case NPC_HORDE_HEALING_WARD:
+            case NPC_TAUREN_WARRIOR:
+            case NPC_HORDE_WITCH_DOCTOR:
+            case NPC_HORDE_PEON:
+            case NPC_THRALL:
+                _baseHorde.insert(creature->GetGUID());
+                break;
+                // Elf base
+            case NPC_DRUID_OF_THE_TALON:
+            case NPC_DRUID_OF_THE_CLAW:
+            case NPC_NELF_ANCIENT_PROT:
+            case NPC_NELF_ANCIENT_OF_LORE:
+            case NPC_NELF_ANCIENT_OF_WAR:
+            case NPC_NELF_ARCHER:
+            case NPC_NELF_HUNTRESS:
+            case NPC_DRYAD:
+            case NPC_TYRANDE:
+                _baseNightElf.insert(creature->GetGUID());
+                break;
 
-                case NPC_INFERNAL_TARGET:
-                    _infernalTargets.push_back(creature->GetGUID());
-                    break;
+            case NPC_INFERNAL_TARGET:
+                _infernalTargets.push_back(creature->GetGUID());
+                break;
 
-                case NPC_WINTERCHILL:
-                case NPC_ANETHERON:
-                case NPC_KAZROGAL:
-                case NPC_AZGALOR:
+            case NPC_WINTERCHILL:
+            case NPC_ANETHERON:
+            case NPC_KAZROGAL:
+            case NPC_AZGALOR:
+            case NPC_NECRO:
+            case NPC_ABOMI:
+            case NPC_GHOUL:
+            case NPC_BANSH:
+            case NPC_CRYPT:
+            case NPC_STALK:
+            case NPC_GARGO:
+            case NPC_FROST:
+            case NPC_INFER:
+                if (_bossWave != TO_BE_DECIDED)
+                    creature->AI()->DoAction(_bossWave);
+                else if (_retreat)
+                    creature->AI()->DoAction(_retreat);
+
+                if (creature->IsSummon() && _bossWave != TO_BE_DECIDED)
+                {
+                    if (_currentWave == 0 && _initialWaves)
+                        creature->SetDisableReputationGain(true);
+                    DoUpdateWorldState(WORLD_STATE_ENEMYCOUNT, ++_trash);    // Update the instance wave count on new trash spawn
+                    _encounterNPCs.insert(creature->GetGUID());             // Used for despawning on wipe
+                }
+                break;
+            case NPC_TOWERING_INFERNAL:
+            case NPC_LESSER_DOOMGUARD:
+            case NPC_DIRE_WOLF:
+            case NPC_GUARDIAN_ELEMENTAL:
+                if (creature->IsSummon())
+                {
+                    _summonedNPCs.insert(creature->GetGUID());
+                }
```

Comment:
remove brackets

Reply:
Sorry. I might be confused, but I did not write that code. This PR is about fixing the trash not cleaning up the code.